ena-submission/src/ena_deposition/ena_submission_helper.py

```diff
@@ -387,8 +387,12 @@ def create_fasta(
         with tempfile.NamedTemporaryFile(delete=False, suffix=".fasta.gz") as temp:
             filename = temp.name
 
+    multi_segment = True
+    if set(unaligned_sequences.keys()) == {"main"}:
+        multi_segment = False
+
```

Comment:
```suggestion
    multi_segment = set(unaligned_sequences.keys()) != {"main"}
```
(not important, feel free to reject)